src/cli/serve/serve.js

```diff
@@ -283,18 +280,11 @@ export default function (program) {
       .option(
         '--run-examples',
         'Adds plugin paths for all the Kibana example plugins and runs with no base path'
-      );
-  }
-
-  if (isServerlessCapableDistribution()) {
```

Comment:
Back to `--serverless` only being available for non-distributables (mind the `if(!isKibanaDistributable())` a few lines above.